src/mmw/apps/user/views.py

```diff
@@ -129,8 +132,6 @@ def itsi_auth(request):
         auth_login(request, user)
         return redirect(request.GET.get('next', '/'))
     else:
-        # User did not authenticate. Save their ITSI ID and send to /sign-up
-        request.session['itsi_id'] = itsi_user['id']
```

Comment:
Removing this since it is actually never used, and also would be lost as the session changes.